charts/cray-certmanager/templates/cert-manager/securityContext-job.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ template "cray-certmanager.fullname" . }}-cert-manager-{{ .Release.Revision }}"
+  labels:
+    app.kubernetes.io/name: {{ template "cray-certmanager.fullname" . }}-cert-manager
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: "cert-manager"
+      containers:
+        - name: "cert-manager"
+          image: "artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15"
```

Comment:
The image to use should be configurable in values.yaml.

Reply:
By convention, the values.yaml parameters should be:

```
kubectl:
  image:
    repository: artifactory.algol60.net/csm-docker/stable/docker-kubectl
    tag: 1.19.15
    pullPolicy: IfNotPresent
```

which means this line would change to:

```
image: {{ .Values.kubectl.image.repository }}:{{ .Values.kubectl.image.tag }}
imagePullPolicy: {{ .Values.kubectl.image.pullPolicy }}
```